### ISSUE TITLE
Kindly give me a hint to resolve

### ISSUE THREAD

**Issue by abbdulwahab86** [spyder-ide/spyder#6150]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\__init__.py", line 110, in eventFilter
    self.tab_moved(event)
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\__init__.py", line 146, in tab_moved
    self.move_tab(from_index, to_index)
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\__init__.py", line 158, in move_tab
    from_plugin = self._get_plugin(from_index)
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\__init__.py", line 62, in _get_plugin
    tab_text = self.dock_tabbar.tabText(index).replace('&', '')
TypeError: tabText(self, int): argument 1 has unexpected type 'NoneType'

## Version and main components

* Spyder Version: 3.2.3 
* Python Version: 3.6.2
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.5.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
numpy >=1.7      :  1.13.1 (OK)
sphinx >=0.6.6   :  1.6.3 (OK)
rope >=0.9.4     :  0.10.5 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0

**Comment by ccordoba12** [comment]
Please describe step by step how you got this error so we can reproduce it in our side. If you fail to do that, we'll close this issue in a week from now.

**Comment by CAM-Gerlach** [comment]
Thanks for reporting; closed as a duplicate of #5813 . Please see that thread for more, thanks. In the meantime, not moving around plugin tabs should avoid it, at least.
